scripts/ci.sh

```diff
@@ -1,55 +1,30 @@
 #!/bin/bash
```

Comment:
Quite a lot of changes in this file, but that goes some way to illustrating the benefit of this PR 😅 .